### DISCUSSION TITLE
Warm start jobs keep getting stopped due to checksum mismatch

### DISCUSSION THREAD

**Discussion by xtian15** [ufs-community/ufs-weather-model#2012] [Q&A]
Hi,
I am running warm start UFS jobs with restart files. But the jobs kept getting terminated with errors like
```
 The checksum in the file:INPUT/sfc_data.tile4.nc and variable:
tsea does not match the checksum calculated from the data. file:E89EA9AC767E277B fr
om data:A2BFF5647BD3173D
```
, even though I have turned off all checksum related checks in the input.nml. I wonder what I am missing and how to get rid of this issue. 

Thanks in advance.

**Comment by jkbk2004** [comment]
@xtian15 checksum related PR was closed at https://github.com/ufs-community/ufs-weather-model/pull/1275. I am not sure if it's setup or input file related. Which machine are you running on?

**Comment by gspetro-NOAA** [comment]
Hi @xtian15,

Did the linked PR clarify things for you? If not, we will need more information about the problem. You can view the [posting guidelines here](https://github.com/ufs-community/ufs-weather-model/discussions/1897) for what additional information to include. We're happy to assist further if you can provide some of that information. 

Thanks,
Gillian Petro

**Reply by xtian15** [reply]
Yes, the issue was resolved. Thanks for the information!
